ccw.core/src/clj/ccw/leiningen/classpath_container.clj

```diff
@@ -40,6 +41,13 @@
 (def LEININGEN_CLASSPATH_CONTAINER_PROBLEM_MARKER_TYPE
   "ccw.leiningen.problemmarkers.classpathcontainer")
 
+(defonce ^{:private true
+           :author "Andrea Richiardi"
+           :doc "Atom containing the state of this namespace.
```

Comment:
Red lights flashing, red lights flashing when talking about the "state of a namespace" as if a namespace was a class ...